### ISSUE TITLE
How to set up with Poetry?

### ISSUE THREAD

**Issue by danieleades** [nipunn1313/mypy-protobuf#347]
How would one go about setting up a [Poetry](https://python-poetry.org/) project to use this plugin?

- `mypy_protobuf` is installed in an isolated environment, so plugin is not on system path
- specifying absolute path to plugin is not portable, since the location of the isolated environment is configurable

**Comment by danieleades** [comment]
thanks for the assist @nipunn1313 

in the end i discovered that protoc has upstreamed something very similar to this library. it's only available on master right now, but protoc will now happily spit out typing files

**Comment by nipunn1313** [comment]
Yep seems to have been published just this week. Let me know how it goes!
